Ryujinx.Common/SystemInfo/WindowsSystemInfo.cs

```diff
@@ -46,7 +46,7 @@ private static string GetCpuNameWMI()
         }
 
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
```

Comment:
```suggestion
        [StructLayout(LayoutKind.Sequential)]
```

This is not needed, let's remove it while we are at it.